src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -13,8 +13,11 @@
 #    include <pwd.h>
 #    if defined(__FreeBSD__)
 #        include <stddef.h>
-#        include <sys/sysctl.h>
+// clang format off
```

Comment:
it's `clang-format`, not `clang format`

Reply:
That's on me, I suggested the latter. Sorry.